src/main/java/codeu/model/data/PasswordUtils.java

```diff
@@ -3,10 +3,14 @@
 /** small class that manages password restrictions **/
 public class PasswordUtils {
 
-    public PasswordUtils() {
+    private PasswordUtils() {
     }
 
-    public boolean isPasswordCorrect(String password){
+    public static boolean isPasswordCorrect(String password){
+        if (password.equals(null)){
```

Comment:
Ah, close, but unfortunately this won't work. If password is null you'll get a NullPointerException .. you need to do

if (password == null) {
  return false;
}

Reply:
oh okay, I just changed it. How come == works but .equals doesn't?

Reply:
well, because .equals() is a method called on the object. If the object is null it will provoke a NullPointerException just like any other method.
== is a java operator, it works if one side or the other is null.

(Be sure to get that right in your interview ;) )